README.md

```diff
@@ -1 +1,82 @@
-# ra-auth0
+# ra-auth-auth0
+
+An auth provider for [react-admin](https://github.com/marmelab/react-admin) which handles authentication via a [Auth0](https://auth0.com) instance.
+
+This repository contains:
+
+-   The actual `ra-auth-auth0` package
+-   A simple demo app you can run locally to try out `ra-auth-auth0` with your own Auth0 instance
+
+## The `ra-auth-auth0` package
+
+-   Please have a look at the [DOCUMENTATION](./packages/ra-auth-auth0/Readme.md)
+-   And also why not the [source code](https://github.com/marmelab/ra-auth-auth0/tree/main/src/packages/ra-auth-auth0)
+
+## Simple Demo
+
+### Prerequesites
+
+-   You need to have a [Auth0](https://auth0.com) account
+
+### Initial setup
+
+1. Clone this project
+
+We need to add some minimal configuration to our Auth0 instance to use it. This need to be done from the Auth0 Admin Console.
+
+1. Go to the [Auth0 Dashboard](https://manage.auth0.com/dashboard)
+1. Select your Auth0 Application (or create a new one)
+1. Add `admin@marmelab.com` user to your Auth0 Application. For this, you need to go to the `User Management` section and create a new user.
+1. Add `admin` to the `Roles` section of your Auth0 Application. Assign admin@marmelab.com to the `admin` role.
+1. Do the same for `user@marmelab.com` and the `user` role.
+1. Go to the `Applications` section of your Auth0 Application and select `Settings`
+1. Create a `single page` application with the following settings:
+    - Allowed Callback URLs: `http://localhost:8081`
+    - Allowed Logout URLs: `http://localhost:8081`
+    - Allowed Web Origins: `http://localhost:8081`
+    - Allowed Origins (CORS): `http://localhost:8081`
```

Comment:
Addresses are `http://127.0.0.1:8081` if your run from Docker.
Also the callback URL is wrong.

```suggestion
    - Allowed Callback URLs: `http://127.0.0.1:8081/auth-callback`
    - Allowed Logout URLs: `http://127.0.0.1:8081`
    - Allowed Web Origins: `http://127.0.0.1:8081`
    - Allowed Origins (CORS): `http://127.0.0.1:8081`
```